### ISSUE TITLE
🚀 [Feature] Allow admins to toggle and edit Donations Bar

### ISSUE THREAD

**Issue by datenbrei** [Human-Connection/Human-Connection#2401]
## :rocket: Feature
We do not any longer ask users about donations like in a marathon. So the donations bar should be replaced by a freeform html field to be flexible about asking for donations. Changes:

- [x] Make the bar disappear

Later:
- [ ] Make the bar in the top of the network be swichable off
- ~[ ] Make the text above the donations bar freely configurable~
- ~[ ] Add a second alternative option for a free form html field for donations info, also switchable off.~

Current situation:
The bar is available although we do not ask for single donation in a marathon any longer
![spendenbalken](https://user-images.githubusercontent.com/13457963/69973441-cd434d00-1523-11ea-8301-f3536b680e39.png)

### User Problem
No donations marathon exists any longer.

### Implementation
The bar should be willingly be replacable by a freeform field which can be set as html in the admin panel, so that we are flexible for the future. 

### Design & Layout
The field's input should be shown up as text and should be responsive. we don't think of something like an article, but more of a one-liner. 

### Validation
If the bar is hidable the current problem is solved. 

### Additional context
There is still a wish to ask for donations, but we should start to get flexible.

**Comment by DennisHack** [comment]
ACHTUNG: ATTENTION: 
Following is important on that issue NOW: 
Amount of people who donate monthly and the amount of money they donate monthly together. 

Therefore we need a nice solution. :-)

**Comment by alina-beck** [comment]
Maybe I missed it because I wasn't in the Friday meeting but could you explain _why_ we don't want the donation progress bar any longer? I'm wondering because it seemed to be so very important just a few weeks ago. What changed?

> Make the bar in the top of the network be swichable off

✅ The on-off toggle is easy to implement.


> Make the text above the donations bar freely configurable

🚫 This does not work with translations. We could maybe add a set of different titles to the translation files and let admins choose between them.


> Add a second alternative option for a free form html field for donations info, also switchable off

❓ I don't know what this means @datenbrei – you're not suggesting that admins can write their own HTML, are you?

**Comment by datenbrei** [comment]
@alina-beck We no longer want to stress people aboout helping us to solve our financial situation. Our project is consequently donation based, but from our point of view it is not good style to ask for monthly donations for ever. 

@ogerly From my point ov view we could close this issue?!?

**Comment by alina-beck** [comment]
@datenbrei I don't think we should – but I renamed it to better reflect what the next steps are about. So far we only implemented step 1 (hide the bar completely) but we have several steps planned and documented in this issue that we can tackle in the future.
